### PR TITLE
Add DiscordKillFeed plugin

### PR DESCRIPTION
**Pull request by LeventHAN**
All changes have been tested, after waiting for 30 minutes on full server I never got a rateLimit (which did suprise me).

After looking the discord docs I have seen that the ratelimit for sending to DiscordAPI is 50req/s.

Fixing this: https://github.com/Thomas-Smyth/SquadJS/issues/197

### CODE REVIEW

**Comment by LeventHAN** [comment]
Hmm after 30 minutes, just now, there was a peak moment where a lot of wounds happend. Which basicly did try to send 6-7 wounds at once/same moment.
This let to hit the Discord's ratelimits.
Happened twice. 
Rate limit hit 4185
Rate limit hit 4298

But I believe the embeds are getting sended anyways... Not 100% sure tho.

**Comment by LeventHAN** [comment]
As a side note;
Apparently 50mes/s is the global rate limit. On server limit it is **5mes/s** 😭

**Comment by LeventHAN** [comment]
DiscordJS is apparently dealing with it and should not be a big problem.

File: squad-server/plugins/discord-killfeed.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+export default class DiscordKillFeed extends DiscordBasePlugin {
+  static get description() {
+    return (
+      'The <code>DiscordKillFeed</code> plugin logs all wounds and related information to a Discord channel for ' +
+      'admins to review.'
+    );
+  }
+
+  static get defaultEnabled() {
+    return true;
```
Comment (Thomas-Smyth) [inline squad-server/plugins/discord-killfeed.js]: I doubt many people will want to use this plugin, I would suggest setting the default to `false`.

**Review by Thomas-Smyth — CHANGES_REQUESTED** [review]

File: squad-server/plugins/discord-killfeed.js (hunk 1 above)
Comment (LeventHAN) [reply squad-server/plugins/discord-killfeed.js]: Done 👍

**Pull request merged**